javascript/401-code-challenges/trees/README.md

```diff
@@ -44,7 +44,7 @@ Space: Add method is O(1), contains method is O(h)
 
 findMaxValue
 Time: O(n)
-Space: O(n)
+Space: O(1)
```

Comment:
Since you are using a recursive method, your call stack means that the space will be O(h).